[2024-02-10T04:18:20.420+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.load_data_to_postgres backfill__2024-02-09T19:00:00+00:00 [queued]>
[2024-02-10T04:18:20.512+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.load_data_to_postgres backfill__2024-02-09T19:00:00+00:00 [queued]>
[2024-02-10T04:18:20.523+0000] {taskinstance.py:2171} INFO - Starting attempt 2 of 2
[2024-02-10T04:18:20.585+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): load_data_to_postgres> on 2024-02-09 19:00:00+00:00
[2024-02-10T04:18:20.605+0000] {standard_task_runner.py:60} INFO - Started process 6423 to run task
[2024-02-10T04:18:20.625+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'load_data_to_postgres', 'backfill__2024-02-09T19:00:00+00:00', '--job-id', '404', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/weather_api__dag.py', '--cfg-path', '/tmp/tmpk9cik9d7']
[2024-02-10T04:18:20.731+0000] {standard_task_runner.py:88} INFO - Job 404: Subtask load_data_to_postgres
[2024-02-10T04:18:21.116+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_api_dag.load_data_to_postgres backfill__2024-02-09T19:00:00+00:00 [running]> on host c5d03f501d38
[2024-02-10T04:18:22.259+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ahaonezero@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='load_data_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-02-09T19:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='backfill__2024-02-09T19:00:00+00:00'
[2024-02-10T04:18:22.326+0000] {logging_mixin.py:188} INFO - /opt/***/dags/weather_data2024-02-09T19:00:00+00:00.json
[2024-02-10T04:18:22.327+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-10T04:18:22.380+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=load_data_to_postgres, execution_date=20240209T190000, start_date=20240210T041820, end_date=20240210T041822
[2024-02-10T04:18:22.671+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-10T04:18:22.755+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
